core/src/main/java/com/scalar/db/api/ScanBuilder.java

```diff
@@ -90,6 +91,11 @@ public BuildableScan projections(Collection<String> projections) {
       return this;
     }
 
+    @Override
+    public BuildableScan projections(String... projections) {
+      return projections(Arrays.asList(projections));
+    }
+
```

Comment:
I think we need to override this method in `Scan.BuildableScanOrScanAllFromExisting` to make it return a `Scan.BuildableScanOrScanAllFromExisting` object.

Reply:
Fixed in the latest commit.